### PR TITLE
New Weights for All Pallets

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR updates all the Weights for the FRAME Pallets.

### CODE REVIEW

**Review by kianenigma — APPROVED** [review]

File: frame/contracts/src/weights.rs [hunk 1]
```diff
@@ -154,1339 +152,1311 @@ pub trait WeightInfo {
...
-		(149_701_000 as Weight)
-			// Standard Error: 297_000
-			.saturating_add((357_149_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
-	}
-	fn seal_rent_status(r: u32, ) -> Weight {
-		(146_863_000 as Weight)
-			// Standard Error: 191_000
-			.saturating_add((638_683_000 as Weight).saturating_mul(r as Weight))
```
Comment (shawntabrizi) [inline frame/contracts/src/weights.rs:964]: @athei where did these benchmarks go?

File: frame/election-provider-multi-phase/src/weights.rs [hunk 2]
```diff
@@ -57,101 +57,105 @@ pub trait WeightInfo {
...
+			// Standard Error: 12_000
+			.saturating_add((3_480_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 42_000
+			.saturating_add((194_000 as Weight).saturating_mul(t as Weight))
+			// Standard Error: 12_000
+			.saturating_add((10_498_000 as Weight).saturating_mul(a as Weight))
+			// Standard Error: 63_000
+			.saturating_add((3_074_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```
Comment (shawntabrizi) [inline frame/election-provider-multi-phase/src/weights.rs:94]: Weights were not updated when making these changes?

cc @coriolinus @kianenigma

File: frame/timestamp/src/weights.rs [hunk 3]
```diff
@@ -45,36 +46,29 @@ use sp_std::marker::PhantomData;
...
-	
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	fn set() -> Weight {
-		(11_650_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
+		(10_277_000 as Weight)
+			.saturating_add(RocksDbWeight::get().reads(3 as Weight))
```
Comment (shawntabrizi) [inline frame/timestamp/src/weights.rs:68]: When did this change happen?

**Comment by shawntabrizi** [comment]
Everything else looks good

File: frame/contracts/src/weights.rs (hunk 1 above)
Comment (athei) [reply frame/contracts/src/weights.rs:964]: Those two are functions marked as unstable. I removed the benchmarks for now but they are re-added once the API is stabilized (or before that). Unstable APIs cannot be used in production chains and are only available when you supply the`unstable-interface` feature to pallet_contracts.

So this is OK.

**Review by athei — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**